tests/tx_cache.py

```diff
@@ -107,7 +107,9 @@ def cli(tx, coin_name):
 
     click.echo(f"Fetching from {tx_url}...")
     try:
-        tx_src = requests.get(tx_url).json(parse_float=Decimal)
+        tx_src = requests.get(tx_url, headers={"user-agent": "x"}).json(
```

Comment:
Would you add a short comment on why this is necessary? Something like blockbook servers refuses requests with empty user agent.

Reply:
https://github.com/trezor/trezor-firmware/pull/1965/commits/ca7ff4cdf4518ab201b943cddee527ee74779cb1